javascript.html.markdown

```diff
@@ -603,31 +603,39 @@ const pi = 3.14;
 
 pi = 4.13; // You cannot do this.
 
-// There is a new syntax for functions in ES6 known as "lambda syntax".
+// There is a new syntax for functions in ES6 known as "arrow function"
+// (lambda syntax in many others languages).
 // This allows functions to be defined in a lexical scope like with variables
 // defined by const and let. 
```

Comment:
I don't think this is true. I think you're conflating the concepts of "function expression" with "arrow function syntax".

We can have function **declarations**:
```
function add(a, b) {
  return a + b;
}
```

There are function **expressions**, in two flavors, both of which are expressions generating functions that can be assigned.
```
const add = function (a, b) {
  return a + b;
};
```

```
const add = (a, b) => a + b;
```